parser/Cargo.toml

```diff
@@ -13,18 +13,16 @@ documentation = "https://docs.rs/gluon"
 build = "build.rs"
 
 [dependencies]
-combine-language = "=2.0.0-beta4"
-combine = "=2.0.0-beta3"
+combine = "2.0.0"
 env_logger = { version = "0.3.4", optional = true }
 quick-error = "1.0.0"
-lalrpop = { git ="https://github.com/Marwes/lalrpop", path = "lalrpop", branch = "symbol_error" }
-lalrpop-util = "0.12.0"
+lalrpop-util = "0.12.2"
 log = "0.3.6"
 gluon_base = { path = "../base", version = "0.2.2" }
 collect-mac = "0.1.0"
 
 [build-dependencies]
-lalrpop = { git ="https://github.com/Marwes/lalrpop", path = "lalrpop", branch = "symbol_error" }
```

Comment:
Before we merge maybe we should rebase any commits with the symbol_error branch to point to crates.io (for easier bisecting later).